vHunter/workers/notify_aggregator_worker.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+import aiohttp
+import logging
+import socket
+from singleton_decorator import singleton
+
+from vHunter.utils import async_every
+from vHunter.notifiers import *  # noqa: F401,F403
+
+
+@singleton
+class NotifyAggregatorWorker:
+    def __init__(self, slave=False, master_host=None, master_port=None):
+        self.slave = slave
+        self.master_host = master_host
+        self.master_port = master_port
+        self.things_to_send = {}
+        self.ioloop = asyncio.get_event_loop()
+        self.ioloop.call_soon(asyncio.ensure_future, self.run())
+
+    def agregate(self, notifier_class, receivers, vulnerabilities):
+        for receiver in receivers:
+            if self.things_to_send.get((notifier_class, receiver)) is None:
+                self.things_to_send[(notifier_class, receiver)] = []
+            self.things_to_send[(notifier_class, receiver)].append(vulnerabilities)
+
+    def load_notifier(self, name):
```

Comment:
This one could be replaced by more generic function from utils module, mentioned before